### ISSUE TITLE
Cannot install with pip: Preparing metadata (pyproject.toml) ... error

### ISSUE THREAD

**Issue by eszter007** [kha-white/mokuro#115]
When I tried `pip3 install mokuro` I got the following error:
```
Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [37 lines of output]
      <string>:26: DeprecationWarning: Use shutil.which instead of find_executable
      fatal: not a git repository (or any of the parent directories): .git
      fatal: not a git repository (or any of the parent directories): .git
      Traceback (most recent call last):
        File "/Users/eszter/japanese/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
          ~~~~^^
        File "/Users/eszter/japanese/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
        File "/Users/eszter/japanese/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 149, in prepare_metadata_for_build_wheel
          return hook(metadata_directory, config_settings)
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 377, in prepare_metadata_for_build_wheel
          self.run_setup()
          ~~~~~~~~~~~~~~^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 522, in run_setup
          super().run_setup(setup_script=setup_script)
          ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 320, in run_setup
          exec(code, locals())
          ~~~~^^^^^^^^^^^^^^^^
        File "<string>", line 271, in <module>
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/__init__.py", line 117, in setup
          return distutils.core.setup(**attrs)
                 ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/_distutils/core.py", line 145, in setup
          _setup_distribution = dist = klass(attrs)
                                       ~~~~~^^^^^^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 330, in __init__
          self.metadata.version = self._normalize_version(self.metadata.version)
                                  ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 366, in _normalize_version
          normalized = str(Version(version))
                           ~~~~~~~^^^^^^^^^
        File "/private/var/folders/xn/vlt8mygs5f3cb6t78m3jf0f40000gn/T/pip-build-env-km6cqk45/overlay/lib/python3.13/site-packages/setuptools/_vendor/packaging/version.py", line 202, in __init__
          raise InvalidVersion(f"Invalid version: {version!r}")
      packaging.version.InvalidVersion: Invalid version: 'unknown'
      [end of output]
```

My system:
- macOS version: 15.2 (24C101) ➔ Intel
- Python 3.13.1

**Comment by eszter007** [comment]
The python version was too high. I fixed it by using python 3.10 instead

**Comment by shizzic** [comment]
> The python version was too high. I fixed it by using python 3.10 instead

THANK YOU!
